### DISCUSSION TITLE
✘ Invalid config override 'ner-d2': name should start with --

### DISCUSSION THREAD

**Discussion by Zanejins** [explosion/spaCy#12354] [Help: Other Questions] [answered]
I use this command to label data:
`python -m prodigy ner.manual ner-d2 blank:en ./merge_RCM.txt --label MD,GD,DD,RD,PHON`

then, I want to train  by this command:
`python -m prodigy train ner ner-d2 blank:en --output ner-model`

But I met the error:
✘ Invalid config override 'ner-d2': name should start with --

I guess because the name has the char '-'. I don't want to label the document again. 

May you tell me how to change the name ? or how to solve this error?
Thank you!

**Comment by adrianeboyd** [comment, marked as answer]
This question would be a better fit for the prodigy support forum: https://support.prodi.gy
